spark-jobs/src/main/scala/za/co/absa/enceladus/common/plugin/menas/EventListenerMenas.scala

```diff
@@ -139,6 +139,10 @@ class EventListenerMenas(config: Config,
     }
   }
 
+  def close(): Unit = {
+    controlMetricPlugins.foreach(plugin => plugin.close())
```

Comment:
Wouldn't it be safer to call the `close` within a `try` block?

Reply:
Agreed. We should be protective when running external code.

Reply:
Actually if you will do it, will you do it for close only, or all outer calls?

Reply:
All other plugins (`factory.apply()` and `plugin.onCheckpoint()`) calls are already protected.